Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -2274,9 +2274,15 @@ private void AddAvailableDevicesThatAreNowRecognized()
                 var layout = TryFindMatchingControlLayout(ref m_AvailableDevices[i].description, id);
                 if (IsDeviceLayoutMarkedAsSupportedInSettings(layout))
                 {
-                    AddDevice(m_AvailableDevices[i].description, false,
-                        deviceId: id,
-                        deviceFlags: m_AvailableDevices[i].isNative ? InputDevice.DeviceFlags.Native : 0);
+                    try
+                    {
+                        AddDevice(m_AvailableDevices[i].description, false,
+                            deviceId: id,
+                            deviceFlags: m_AvailableDevices[i].isNative ? InputDevice.DeviceFlags.Native : 0);
+                    }
+                    catch (Exception)
+                    {
+                    }
```

Comment:
This is an unrelated tweak to a change I just landed. Some HID devices led to exception spam in the console. The code here should just go and ignore those devices.